README.md

```diff
@@ -51,6 +51,25 @@ tree.Sorter = func(u1, u2 fyne.URI) bool {
 </p>
 
 
+### 7-Segment ("Hex") Display
+
+A skeuomorphic widget simulating a 7-segment "hex" display. Support setting
```

Comment:
Shouldn’t it be “Supports” with an “s” at the end?

Reply:
6411fe6